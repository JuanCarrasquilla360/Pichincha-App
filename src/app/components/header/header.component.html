<header>
  <div class="logo">
    <a href="/">
      <img src="assets/img/logo-header.jpg">
    </a>
  </div>
</header>