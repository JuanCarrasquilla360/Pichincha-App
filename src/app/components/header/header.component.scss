header {
    background-color: #FFF;
    height: 100px;
    display: flex;
    align-items: center;
    justify-content: center;

}

.logo img {
    height: 60px;
}