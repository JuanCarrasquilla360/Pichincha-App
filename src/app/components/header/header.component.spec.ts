import { ComponentFixture, TestBed } from '@angular/core/testing';

import { HeaderComponent } from './header.component';

describe('HeaderComponent', () => {
  let component: HeaderComponent;
  let fixture: ComponentFixture<HeaderComponent>;
  let compiled: HTMLElement;
  

  beforeEach(async () => {
    await TestBed.configureTestingModule({
      imports: [HeaderComponent]
    })
    .compileComponents();
    
    fixture = TestBed.createComponent(HeaderComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
    compiled = fixture.nativeElement;
  });

  test('should create', () => {
    expect(component).toBeTruthy();
  });
  test('should match with snapshot', () => {
    expect(compiled).toMatchSnapshot();
  });
});
