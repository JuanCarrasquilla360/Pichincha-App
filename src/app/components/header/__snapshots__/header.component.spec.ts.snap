// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderComponent should match with snapshot 1`] = `
<div
  id="root1"
>
  <header>
    <div
      class="logo"
    >
      <a
        href="/"
      >
        <img
          src="assets/img/logo-header.jpg"
        />
      </a>
    </div>
  </header>
</div>
`;
